src/test/java/co/rsk/federate/BtcToRskClientTest.java

```diff
@@ -2534,10 +2534,7 @@ void updateBridge_noUpdateBridgeConfigDefined_shouldOnlyCallSendUpdateCollection
 
         btcToRskClient.updateBridge();
 
-        verify(btcToRskClient, times(0)).updateBridgeBtcBlockchain();
-        verify(btcToRskClient, times(0)).updateBridgeBtcCoinbaseTransactions();
-        verify(btcToRskClient, times(0)).updateBridgeBtcTransactions();
-        verify(federatorSupport, times(1)).sendUpdateCollections();
+        verify(federatorSupport).sendUpdateCollections();
```

Comment:
What about the other assertions? It should call all of them

Reply:
Updated